samtranslator/model/intrinsics.py

```diff
@@ -88,7 +92,7 @@ def make_combined_condition(
     :param string condition_name: base name desired for new condition
     :return: dictionary of condition_name: condition_value
     """
-    if len(conditions_list) < 2:
+    if len(conditions_list) < MIN_NUM_CONDITIONS_TO_COMBINE:
         # Can't make a condition if <2 conditions provided.
```

Comment:
Similar nit about hard coded number in comment.

Reply:
updated to "enough"